### PR TITLE
Delete applicant command

### PR DESCRIPTION
**Pull request by e0543517**

### CODE REVIEW

**Review by e0543517 — COMMENTED** [review]
Todo: Change to use index to access the list instead of id.

File: src/main/java/seedu/address/logic/commands/applicant/DeleteApplicantCommand.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.applicant.Applicant;
+
+/**
+ * Constructs a delete applicant command.
+ */
+public class DeleteApplicantCommand extends Command {
+    public static final String COMMAND_WORD = "deleteapplicant";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
```
Comment (seanleong339) [inline src/main/java/seedu/address/logic/commands/applicant/DeleteApplicantCommand.java:21]: Change to index?

**Review by seanleong339 — CHANGES_REQUESTED** [review]
Minor change to MESSAGE_USAGE required, everything else LGTM

File: src/main/java/seedu/address/logic/commands/applicant/DeleteApplicantCommand.java (hunk 1 above)
Comment (e0543517) [reply src/main/java/seedu/address/logic/commands/applicant/DeleteApplicantCommand.java:21]: Done.

**Review by seanleong339 — APPROVED** [review]
LGTM

**Pull request merged**